source/manual/emergency-publishing.html.md

```diff
@@ -211,7 +211,7 @@ You can manually check whether the data has been stored in Redis by the Jenkins
 From your development machine SSH into a frontend box appropriate to the environment you want to check
 
 ```
-ssh frontend1.frontend.integration
+ssh frontend-1.frontend.integration
```

Comment:
Well... this won't work either, given the changes to AWS?

Reply:
Or rather, given the move to AWS.

Reply:
@cbaines True. I'll change the original command to point to staging and see if i can figure out how to run it on integration.

Reply:
@leenagupte - if you want it to point specifically to integration, one should be able to do something like 

```
ssh integration -At "ssh -q \`govuk_node_list --single-node -c frontend\`"
```

to get straight on to an integration box

Or you could go for `govukcli` https://github.com/alphagov/govuk-aws/blob/master/doc/guides/connecting-to-the-environment.md

Reply:
@bevanloon @cbaines I've updated the command for integration.